recipes/astropy-healpix/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "astropy-healpix" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "99831e3f8fd8af4452e5ba9af27b6ecdbc65a899ed7213ecb9f59671eb616659" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt --offline --no-git
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy 1.10.*  # [not (win and (py35 or py36))]
+    - numpy 1.11.*  # [win and (py35 or py36)]
+  run:
+    - python
+    - numpy >=1.10  # [not (win and (py35 or py36))]
+    - numpy >=1.11  # [win and (py35 or py36)]
+    - astropy >=1.2
+
+test:
+  imports:
+    - astropy_healpix
+    - astropy_healpix.core_cython
+
+about:
+  home: http://github.com/astropy/astropy-healpix
+  license: BSD
```

Comment:
Please update to `BSD 3-clause` in the feedstock so that the version of BSD is clear. Also please re-render after making that change to update the feedstock's README.